docker/build.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+set -e
+set -x
+
+build() {
+    ACTION=${1}
+    PHP_VERSION_TAG=${2}
+    PHP_VERSION_REAL=${3:-${PHP_VERSION_TAG}}
+
+    DOCKER_IMAGE_TAG="ghcr.io/zenstruck/foundry/php:${PHP_VERSION_TAG}"
+
+    # From inside the containers, docker host ip is different in linux and macos
+    UNAME=$(uname -s)
+    if [ "${UNAME}" = "Linux" ]; then
+        XDEBUG_HOST="172.17.0.1"
+    elif [ "${UNAME}" = "Darwin" ]; then
+        XDEBUG_HOST="host.docker.internal"
+    fi
+
+    docker buildx build \
+        --tag "${DOCKER_IMAGE_TAG}" \
+        --cache-from "type=registry,ref=${DOCKER_IMAGE_TAG}" \
+        --build-arg BUILDKIT_INLINE_CACHE=1 \
+        --build-arg PHP_VERSION="${PHP_VERSION_REAL}" \
+        --build-arg XDEBUG_HOST="${XDEBUG_HOST}" \
+        --build-arg UID="$(id -u)" \
+        --file docker/Dockerfile \
+        "--${ACTION}" \
+        .
+}
+
+main() {
+    ACTION="${1:-load}"
+
+    if [ "${ACTION}" != 'push' ] && [ "${ACTION}" != 'load' ]; then
+        echo "Error: action \"${1}\" invalid. Allowed actions are push|load"
+        exit 1;
+    fi
+
+    # ensure default builder is used, it is needed to benefit from cache layering
+    docker buildx use default
+
+    build "${ACTION}" 8.0
+#    build "${ACTION}" 8.1
+#    build "${ACTION}" 8.2 8.2.0RC7
```

Comment:
Don't need the RC anymore!

Reply:
still needed, the official image is not released
https://hub.docker.com/_/php/tags?page=1&name=8.2